sway/desktop/xwayland.c

```diff
@@ -382,30 +382,19 @@ static const struct sway_view_impl view_impl = {
 	.destroy = destroy,
 };
 
-static void get_geometry(struct sway_view *view, struct wlr_box *box) {
```

Comment:
Could you say a few more words about why we're making this change? Not saying it's wrong, I just don't understand from the diff + commit title.

Reply:
We could drop this commit. The existence of view->surface in this old get_geometry is unnecessary and it just turned into a tiny 2 line function that I thought inlining wouldn't be an issue.